vscode-wpilib/src/java/deploydebug.ts

```diff
@@ -203,14 +203,19 @@ class SimulateCodeDeployer implements ICodeDeployer {
           path: e,
         });
       }
-      const quickPick = await vscode.window.showQuickPick(extList, {
-        canPickMany: true,
-        placeHolder: 'Pick extensions to run',
-      });
-      if (quickPick !== undefined) {
-        for (const qp of quickPick) {
-          extensions += qp.path;
-          extensions += path.delimiter;
+      if (this.preferences.getPreferences(workspace).getSelectDefaultSimulateExtension() && extList.length === 1) {
+        extensions += extList[0].path;
+        extensions += path.delimiter;
```

Comment:
I kept in this delimiter at the end because, although unnecessary, it is present at the end of the list when picking extensions manually, and I didn't want to alter this behavior between the two.